database/migrations/2020_12_22_105909_create_webhook_calls_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateWebhookCallsTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('webhook_calls', static function (Blueprint $table): void {
```

Comment:
Is there anything particularly sensitive that will be stored in this table?

Reply:
It will store [`payment.created`](https://developer.squareup.com/reference/square/payments-api/webhooks/payment.created) and [`payment.updated`](https://developer.squareup.com/reference/square/payments-api/webhooks/payment.updated) events. There are lots of fields there that are not super fun to store, but given that nothing in the table will be exposed via Nova or API I think we should be OK?